devicetypes/vision-stevenchen/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -0,0 +1,197 @@
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+
+/**
+ *  Vision Zigbee Arrival Sensor
+ *
+ *  Author: Steven Chen
+ */
+metadata {
+    definition (name: "Arrival Sensor HA", namespace: "vision-stevenchen", author: "Steven Chen",
+            runLocally: true, minHubCoreVersion: '000.025.00032', executeCommandsLocally: true) {
```

Comment:
you'll need to remove the `runLocally`, `minHubCoreVersion`, and `executeCommandsLocally` flags and change the name of the DTH here as well as the file/folder names.

Reply:
OK! I can remove the runLocally , minHubCoreVersion, and executeCommandsLocally flags. But when I change the name of DTH from "Arrival Sensor HA" to "Vision Zigbee Arrival Sensor". I find a bug of SmartThings app. In SmartThings app the tiles represent order change as "battery" "presence" "beep". Original represent order is "presence" "beep" "battery". I only change the name of DTH.  I don't modify the other code and details() definition. Screen as below image.
![IMG_0551](https://user-images.githubusercontent.com/76581149/114968423-e181b300-9ea8-11eb-9de1-531aac525625.PNG)

Reply:
I already solve the issue above.